hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTable.java

```diff
@@ -107,26 +107,24 @@ protected AbstractTable(final UIEventBus eventBus, final VaadinMessageSource i18
     private void onValueChange() {
         eventBus.publish(this, UploadArtifactUIEvent.HIDE_DROP_HINTS);
 
-        final Set<I> values = getTableValue(this);
+        final Set<Long> values = getTableValue(this);
```

Comment:
If the generic `I` is useless, please remove ist complete from the `AbstractTable`  

`public abstract class AbstractTable<E extends NamedEntity> extends Table implements RefreshableContainer`